### ISSUE TITLE
Perplexity issue

### ISSUE THREAD

**Issue by JudeLee19** [suriyadeepan/easy_seq2seq#29]
Even though I didn't edit code when I apporach to training step 13800 than each bucket perplexity is so high like below.

global step 13800 learning rate 0.4851 step-time 0.18 perplexity 1.01
  eval: bucket 0 perplexity 8992400.69
  eval: bucket 1 perplexity 23779421.72
  eval: bucket 2 perplexity 100536278.99
  eval: bucket 3 perplexity 48283674.77

Is it something wrong ?
